src/main/frontend/components/content.cljs

```diff
@@ -168,11 +158,12 @@
         [:.menu-links-wrapper
          [:div.flex-row.flex.justify-between.pb-2.pt-1.px-2
           [:div.flex-row.flex.justify-between
-           (for [color block-background-colors]
+           (for [color ui/block-background-colors]
              [:a.m-2.shadow-sm
-              {:on-click (fn [_e]
+              {:title (t (keyword "color" (:id color)))
+               :on-click (fn [_e]
                            (editor-handler/set-block-property! block-id "background-color" color))}
-              [:div.heading-bg {:style {:background-color color}}]])
+              [:div.heading-bg {:style {:background-color (str "var(--color-" color "-500)")}}]])
```

Comment:
We are using a naming convention based on the `block-background-colors` vector.
A vector of maps that explicitly sets the name of the color and the css properties could be safer and more open, but it feels a little verbose.
```
(def block-background-colors
   [{:id "gray"
     :title (t :color/gray)
     :default-color "var(--color-gray-500)" 
     :highlight-color "var(--ls-highlight-color-gray)"}
   ...])
```